test/test_nn.py

```diff
@@ -41,7 +41,7 @@
     ctcloss_reference, new_module_tests, single_batch_reference_fn, _test_bfloat16_ops, _test_module_empty_input
 from torch.testing._internal.common_device_type import dtypesIfMPS, instantiate_device_type_tests, dtypes, \
     dtypesIfCUDA, precisionOverride, skipCUDAIfCudnnVersionLessThan, onlyCUDA, onlyCPU, \
-    skipCUDAIfRocm, skipCUDAIf, skipCUDAIfNotRocm, \
+    skipCUDAIfRocm, skipCUDAIf, skipCUDAIfNotRocm, skipMPSVersionIfLessThan, \
```

Comment:
@malfet I just saw #134858. Sorry if I broke something. I think that was introduced by me. I guess I shouldn't use `skipMPSVersionIfLessThan`? Can you walk me through why it's bad? Or point me to some place where I can learn why it's discouraged in PyTorch?